### PR TITLE
Add option to paint a picture.

### PR DESCRIPTION
**Pull request by mhadjiantonis**

### CODE REVIEW

**Comment by mhadjiantonis** [comment]
Please check this, @casey-collab

File: english/marshmallow.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ January 2017 approaches. You have two presidential candidates to vote for.
 
 You don't like either of these candidates.
 So instead, you decide to [write a symphony](write-a-symphony/symphony.md)
+or [paint a picture](paint-a-picture.md).
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline english/marshmallow.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/paint-a-picture/paint-a-picture.md [hunk 2]
```diff
@@ -0,0 +1 @@
+With your eyes closed, you paint a perfect replica of Da Vinci's Mona Liza.
```
Comment (casey-collab) [inline english/paint-a-picture/paint-a-picture.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/marshmallow.md [hunk 3]
```diff
@@ -35,6 +35,7 @@ January 2017 approaches. You have two presidential candidates to vote for.
 
 You don't like either of these candidates.
 So instead, you decide to [write a symphony](write-a-symphony/symphony.md)
+or [paint a picture](paint-a-picture/paint-a-picture.md).
```
Comment (casey-collab) [inline english/marshmallow.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by mhadjiantonis** [comment]
How about nor @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by mhadjiantonis** [comment]
I think I fixed everything @casey-collab

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**